{
	"id": "SUSE-SU-2025:0049-1",
	"modified": "2025-01-09T15:37:07Z",
	"published": "2025-01-09T15:37:07Z",
	"related": [
		"CVE-2024-11168",
		"CVE-2024-9287"
	],
	"summary": "Security update for python310",
	"details": "This update for python310 fixes the following issues:\n\n- Update to 3.10.16\n",
	"affected": [
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.6",
				"name": "python310",
				"purl": "pkg:rpm/opensuse/python310&distro=openSUSE%20Leap%2015.6"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.10.16-150400.4.66.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"libpython3_10-1_0": "3.10.16-150400.4.66.1",
						"libpython3_10-1_0-32bit": "3.10.16-150400.4.66.1",
						"python310": "3.10.16-150400.4.66.1",
						"python310-32bit": "3.10.16-150400.4.66.1",
						"python310-base": "3.10.16-150400.4.66.1",
						"python310-base-32bit": "3.10.16-150400.4.66.1",
						"python310-curses": "3.10.16-150400.4.66.1",
						"python310-dbm": "3.10.16-150400.4.66.1",
						"python310-devel": "3.10.16-150400.4.66.1",
						"python310-doc": "3.10.16-150400.4.66.1",
						"python310-doc-devhelp": "3.10.16-150400.4.66.1",
						"python310-idle": "3.10.16-150400.4.66.1",
						"python310-testsuite": "3.10.16-150400.4.66.1",
						"python310-tk": "3.10.16-150400.4.66.1",
						"python310-tools": "3.10.16-150400.4.66.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.6",
				"name": "python310-core",
				"purl": "pkg:rpm/opensuse/python310-core&distro=openSUSE%20Leap%2015.6"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.10.16-150400.4.66.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"libpython3_10-1_0": "3.10.16-150400.4.66.1",
						"libpython3_10-1_0-32bit": "3.10.16-150400.4.66.1",
						"python310": "3.10.16-150400.4.66.1",
						"python310-32bit": "3.10.16-150400.4.66.1",
						"python310-base": "3.10.16-150400.4.66.1",
						"python310-base-32bit": "3.10.16-150400.4.66.1",
						"python310-curses": "3.10.16-150400.4.66.1",
						"python310-dbm": "3.10.16-150400.4.66.1",
						"python310-devel": "3.10.16-150400.4.66.1",
						"python310-doc": "3.10.16-150400.4.66.1",
						"python310-doc-devhelp": "3.10.16-150400.4.66.1",
						"python310-idle": "3.10.16-150400.4.66.1",
						"python310-testsuite": "3.10.16-150400.4.66.1",
						"python310-tk": "3.10.16-150400.4.66.1",
						"python310-tools": "3.10.16-150400.4.66.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.6",
				"name": "python310-documentation",
				"purl": "pkg:rpm/opensuse/python310-documentation&distro=openSUSE%20Leap%2015.6"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.10.16-150400.4.66.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"libpython3_10-1_0": "3.10.16-150400.4.66.1",
						"libpython3_10-1_0-32bit": "3.10.16-150400.4.66.1",
						"python310": "3.10.16-150400.4.66.1",
						"python310-32bit": "3.10.16-150400.4.66.1",
						"python310-base": "3.10.16-150400.4.66.1",
						"python310-base-32bit": "3.10.16-150400.4.66.1",
						"python310-curses": "3.10.16-150400.4.66.1",
						"python310-dbm": "3.10.16-150400.4.66.1",
						"python310-devel": "3.10.16-150400.4.66.1",
						"python310-doc": "3.10.16-150400.4.66.1",
						"python310-doc-devhelp": "3.10.16-150400.4.66.1",
						"python310-idle": "3.10.16-150400.4.66.1",
						"python310-testsuite": "3.10.16-150400.4.66.1",
						"python310-tk": "3.10.16-150400.4.66.1",
						"python310-tools": "3.10.16-150400.4.66.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2025/suse-su-20250049-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1232241"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1233307"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-11168"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-9287"
		}
	]
}
